components/TextFields/examples/TextFieldOutlinedExample.swift

```diff
@@ -20,8 +20,7 @@ import MaterialComponents.MaterialTextFields_TypographyThemer
 final class TextFieldOutlinedSwiftExample: UIViewController {
 
   let scrollView = UIScrollView()
-  @objc var colorScheme = MDCSemanticColorScheme()
-  @objc var typographyScheme = MDCTypographyScheme()
+  @objc var containerScheme = MDCContainerScheme()
```

Comment:
I can't comment on the `import` lines above, but this change should also remove the Themer imports and add one for MDCContainerScheme.

Reply:
Ah thanks